package.json

```diff
@@ -96,7 +93,7 @@
 		"chai": "=4.0.2",
 		"chai-as-promised": "=7.1.1",
 		"chai-bignumber": "=2.0.0",
```

Comment:
Can you double check if we are really using this one? I guess on very few places we use it. If we update those tests then we don't need this dependency.

Reply:
Did not found usages so I removed it.